modules/installation-vsphere-installer-infra-requirements.adoc

```diff
@@ -40,6 +40,19 @@ A user requires the following privileges to install an {product-title} cluster:
 
 For more information about creating an account with only the required privileges, see link:https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.security.doc/GUID-5372F580-5C23-4E9C-8A4E-EF1B4DD9033E.html[vSphere Permissions and User Management Tasks] in the vSphere documentation.
 
+[discrete]
+[id="installation-vsphere-installer-infra-requirements-vmotion_{context}"]
+== Using {product-title} with vMotion
+
+[IMPORTANT]
+====
+{product-title} generally supports compute-only vMotion. Using Storage vMotion can cause issues and is not supported.
+====
+
+If you are using vSphere volumes in your pods, migrating a VM across datastores either manually or through Storage vMotion causes invalid references within {product-title} persistent volume (PV) objects. These references prevent affected pods from starting up and can result in data loss.
+
+Similarly, {product-title} does not support selective migration of VMDKs across datastores, using datastore clusters for VM provisioning or for dynamic or static provisioning of PVs, or using a datastore that is part of a datastore cluster for dynamic or static provisioning of PVs.
+
```

Comment:
Should we provide a way for admins to see if vmotion is enabled or configured, and might/could cause problems? 

cc: @reestr do you know how to do this?

Reply:
We added https://github.com/openshift/vsphere-problem-detector/pull/28 which ships with default OCP-4.7 install to check datastore clusters and warn users about it. Other than that, we also know people can manually migrate VMs with all attached VMDKs but that is not something we can fix or detect easily.

Reply:
Other than that - we also have https://bugzilla.redhat.com/show_bug.cgi?id=1904482 as a follow up to improve on vmotion/manually movement of VMDKs , but this check will only tell user that their PVs have missing VMDKs, it can't tell them why.

Reply:
At the end of this vMotion section, for 4.7+ docs we could add, “For more information about common installation and misconfiguration issues that are related to storage on vSphere clusters, see [Using the vSphere Problem Detector Operator](https://docs.openshift.com/container-platform/4.7/installing/installing_vsphere/using-vsphere-problem-detector-operator.html)”.

Reply:
Yes lets link to this.